AppCenterIdentity/AppCenterIdentity/MSIdentity.m

```diff
@@ -279,6 +298,30 @@ - (MSIdentityConfig *)deserializeData:(NSData *)data {
   return config;
 }
 
+- (BOOL)clearAuthData {
+  if (![[MSAuthTokenContext sharedInstance] clearAuthToken]) {
+    return NO;
+  }
+  [self removeAccount];
```

Comment:
If `clearAuthData` returns a `BOOL`, shouldn't the 3 methods that it calls (`removeAccount`, `removeAuthToken` and `removeAccountId`) also return a `BOOL` and their result be taken into account for the return parameter of `clearAuthData`? From a consistency perspective, it makes sense. Thoughts?

Reply:
@ElektrojungeMS It really looks like a minor refactoring, so I'm going to complete this one. We can do it separately after the discussion.